nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -121,19 +120,22 @@ def get_edge_params(
             edge tensor shape.
         """
         output_port_id = 0
+        tensor_shape = None
         if source_node.op in ("get_attr",):
             tensor_shape = tuple(getattr(model, source_node.target).shape)
         elif "val" in source_node.meta:
             if source_nncf_node.metatype is om.PTBatchNormMetatype:
                 tensor = source_node.meta["val"][0]
```

Comment:
Is is possible when output_idx is not 0?

Reply:
I don't get your question, could you please add some details?